docs/functions.rst

```diff
@@ -44,6 +44,15 @@ hipBLAS function uses the following notations to denote precisions:
 - c  = single complex
 - z  = double complex
 
+bfloat 16 Datatype
+==================
+hipBLAS defines a hipblasBfloat16 datatype. This type is exposed as a struct simply containing 16 bits of data. There is also a C++ hipblasBfloat16 class defined
+which gives slightly more functionality, including conversion to and from a 32-bit float datatype. This class can be used by defining HIPBLAS_BFLOAT16_CLASS before including
+the header file hipblas.h.
```

Comment:
Since we want to clarify version constraints should add C++ version required for exposure of the class along with the define, e.g. C++11 or greater and defining HIPBLAS_BFLOAT16_CLASS , or similar